mopidy/softwaremixer/__init__.py

```diff
@@ -16,6 +16,12 @@ def get_default_config(self):
 
     def get_config_schema(self):
         schema = super().get_config_schema()
+        schema["min_volume"] = config.Integer(minimum=0, maximum=100)
+        schema["max_volume"] = config.Integer(minimum=0, maximum=100)
+        schema["volume_scale"] = config.String(
+            choices=("linear", "exp")
+        )
+        schema["volume_exp"] = config.Float(minimum=1, maximum=3)
```

Comment:
It would be great to have the rationale behind a minimum of `1` and a maximum of `3` documented somewhere.

Reply:
I added a comment in the code, but I assume it should also go in the actual documentation (in a different repo).